x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -43,8 +44,10 @@
             null,
             null
         ));
+    private static final ServiceAccount KIBANA_SYSTEM_ACCOUNT =
+        new ElasticServiceAccount("kibana-system", ReservedRolesStore.kibanaSystemUserRole(NAMESPACE + "/kibana-system"));
```

Comment:
Sorry if it was already discussed elsewhere, but I'm wondering if the name of the service account should be `kibana` instead of `kibana-system`. Given this is a service account, I don't think there may be confusion with a kibana user, and the name sounds clearer to me and more consistent with `fleet-server`.

What do you think?

Reply:
I don't have an opinion tbh, happy bothways. I'll leave this up to you and Kibana folks to decide

Reply:
I was initially wondering why not `kibana_system` (with the underscore) to be consistent with the existing built-in user (I guess Fleet doesn't have any "system" built-in users yet, so it's not a problem for them). At the same time, I see the name format for service accounts is `<namespace>/<service>` where `service` already implies it's not a user, and `elastic` namespace would imply that this service is a reserved/"system" one. With this, `elastic/kibana` sounds to me.

@jportner @kobelb any concerns regarding `elastic/kibana` service account name?

Reply:
The `elastic/fleet-server` account was initially named `elastic/fleet`. It got renamed because the Fleet team wanted to differentiate between the Fleet Server and other fleet services, e.g. there could be another `elastic/fleet` service account that is used to setup Fleet in Kibana. Do we foresee other service accounts needed for Kibana? If not, `elastic/kibana` might be a better choice.

Reply:
> @jportner @kobelb any concerns regarding `elastic/kibana` service account name?

Fine by me

> Do we foresee other service accounts needed for Kibana? If not, `elastic/kibana` might be a better choice.

Looking through the open issues, I don't think there is a desire to make additional service accounts for Kibana. But Perhaps @kobelb knows something I don't, so let's see what he says.

Reply:
> @jportner @kobelb any concerns regarding elastic/kibana service account name?

Fine by me.

> The elastic/fleet-server account was initially named elastic/fleet. It got renamed because the Fleet team wanted to differentiate between the Fleet Server and other fleet services, e.g. there could be another elastic/fleet service account that is used to setup Fleet in Kibana

This makes sense to me because Fleet as a product is partially implemented in Kibana and Fleet Server. However, at the moment,  we don't have the same complexity with Kibana, Kibana is just Kibana at the moment.

> Looking through the open issues, I don't think there is a desire to make additional service accounts for Kibana. But Perhaps @kobelb knows something I don't, so let's see what he says.

TL;DR: I'm not aware of any concrete plans to introduce additional service accounts.

There have been some super preliminary discussions about how we could in the future allow Kibana to run certain setup steps with different privileges than during normal runtime. For example, when Kibana is first starting up, it needs privileges to provision and migrate data in its own system indices; however, these privileges are no longer required after Kibana has started up. If Kibana could drop privileges after performing setup, it could allow Kibana to run normal operations with fewer privileges, mitigating some attack vectors. However, we'd almost certainly want to find a way for end-users to only have to configure a single reserved user or service account in the kibana.yml, or else we'd make configuring Kibana harder.